apps/condo/domains/common/components/helpers/highlited.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { FilterValue } from 'antd/es/table/interface'
+import { Typography } from 'antd'
+import isEmpty from 'lodash/isEmpty'
+
+import { Highliter, THighliterRenderPartFN } from '../Highliter'
```

Comment:
typo: Highli**gh**ter

And also, there's improved version of this component: `@condo/domains/common/components/TextHighlighter`, so this one is redundant I guess